components/testimonials.tsx

```diff
@@ -29,137 +36,93 @@ const testimonialDataMobile = [
-            </Carousel>
-          </div>
-        </div>
+        <span className="quote-character opacity-[0.2]">"</span>
+      </p>
+      <div className="flex items-center gap-4 px-4 py-4 mt-8 text-sm text-white rounded-lg md:text-xl bg-gradient-to-r from-secondary-500">
+        <Image src={image} alt="logo" className="w-auto h-4" />
+        <p className="font-bold">
+          {author} <span className="text-lg md:text-2xl">/</span> {company}
+        </p>
       </div>
+    </div>
+  );
+};
+export default function Testimonials() {
+  const [index, setIndex] = useState(0);
+  const container = useRef(null);
 
-      {/*<div className="max-w-6xl mx-auto px-4 sm:px-6">*/}
-      {/*  <div className="py-12 md:py-20">*/}
-
-      {/*    /!* Testimonials *!/*/}
-      {/*    <div className="max-w-3xl mx-auto mt-20" data-aos="zoom-y-out">*/}
-      {/*      <div className="relative flex items-start border-2 border-gray-200 border-opacity-25 rounded bg-gradient-to-bl from-neutral-300 to-neutral-200 rounded-lg transition duration-300 hover:shadow-xl">*/}
-
-      {/*        /!* Testimonial *!/*/}
-      {/*        <div className="text-center px-12 py-8 pt-20 mx-4 md:mx-0">*/}
-      {/*          <div className="absolute top-0 -mt-8 left-1/2 transform -translate-x-1/2 ">*/}
-      {/*            <svg className="absolute top-0 right-0 -mr-12 w-16 h-16 fill-current text-primary-300" viewBox="0 0 64 64" aria-hidden="true" xmlns="http://www.w3.org/2000/svg">*/}
-      {/*              <path d="M37.89 58.338c-2.648-5.63-3.572-10.045-2.774-13.249.8-3.203 8.711-13.383 23.737-30.538l2.135.532c-6.552 10.033-10.532 17.87-11.939 23.515-.583 2.34.22 6.158 2.41 11.457l-13.57 8.283zm-26.963-6.56c-2.648-5.63-3.572-10.046-2.773-13.25.799-3.203 8.71-13.382 23.736-30.538l2.136.533c-6.552 10.032-10.532 17.87-11.94 23.515-.583 2.339.22 6.158 2.41 11.456l-13.57 8.283z" />*/}
-      {/*            </svg>*/}
-      {/*            <Image className="relative " src={TestimonialImage} width={196} height={196} alt="Maruti Suzuki Testimonial" />*/}
-      {/*          </div>*/}
-      {/*          <blockquote className="text-xl font-medium mb-4 text-accent-200 italic">*/}
-      {/*            “ Our recent collaboration with Keploy was truly remarkable. <br/> Keploy team provided exceptional support and remained highly proactive throughout the entire experience.*/}
-      {/*              <br/> Keploy platform has added business value to Maruti Suzuki “*/}
-      {/*          </blockquote>*/}
-      {/*          <cite className="block font-bold text-lg not-italic mb-1 text-primary-300">Executive Member</cite>*/}
-      {/*          <div className="text-gray-600 text-seconday-300">@Maruti Suzuki</div>*/}
-      {/*        </div>*/}
-      {/*      </div>*/}
-      {/*    </div>*/}
+  useGSAP(
+    () => {
+      gsap.to('.quote-character', {
+        scrollTrigger: {
+          trigger: '.quote-container',
+          start: 'top bottom',
+          end: 'bottom+=500 bottom',
```

Comment:
We can reduce the end of scroll trigger as right now even when the whole section is visible the text is opaque 
![Screenshot from 2024-03-20 23-51-55](https://github.com/keploy/website/assets/78696767/b29d12d6-2f32-47c1-bcc4-72664f026221)